acc-testcases/data-sources/nsx-tier1.yaml

```diff
@@ -0,0 +1,3 @@
+acc:
+- config: |
+    name = "nsx_tier1_gateway"
```

Comment:
same here

Reply:
add a validation to see if the provider_id is returned here.

Reply:
done